src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/RuntimeEventSource.cs

```diff
@@ -66,6 +68,12 @@ internal void LogAppContextSwitch(string switchName, int value)
             base.WriteEvent((int)EventId.AppContextSwitch, switchName, value);
         }
 
+        [Event((int)EventId.ProcessorCount, Level = EventLevel.Informational, Keywords = Keywords.NumProcesses)]
+        internal void ProcessorCount(int processorCount)
```

Comment:
What calls this when?

Reply:
I assume the PR is incomplete. I'd recommend we call it inside OnEventCommand() when the EventSource is being enabled, similar to the LogSwitchValues() down on line 115.